lighthouse-core/audits/screenshot-thumbnails.js

```diff
@@ -79,16 +78,7 @@ class ScreenshotThumbnails extends Audit {
     const speedline = await Speedline.request(trace, context);
 
     // Make the minimum time range 3s so sites that load super quickly don't get a single screenshot
-    let minimumTimelineDuration = context.options.minimumTimelineDuration || 3000;
-    // Ensure thumbnails cover the full range of the trace (TTI can be later than visually complete)
-    if (context.settings.throttlingMethod !== 'simulate') {
-      const devtoolsLog = artifacts.devtoolsLogs[Audit.DEFAULT_PASS];
-      const metricComputationData = {trace, devtoolsLog, settings: context.settings};
-      const tti = Interactive.request(metricComputationData, context);
-      try {
-        minimumTimelineDuration = Math.max((await tti).timing, minimumTimelineDuration);
-      } catch (_) {}
-    }
```

Comment:
Why don't we need this anymore?

Reply:
we *could* keep it and just fork the behavior in screenshot-thumbnails for timespans, but honestly in practice the past few years I've felt that this tends to devolve the screenshot thumbnails into being 9 copies of the exact same final frame.

I wasn't particularly interested in the extra dance to keep this bit around and viewed it as a slight negative behavior anyway, so I just removed it instead of fixing it. Open to doing the work to support both if folks feel differently!

Reply:
I don't have an attachment to it, just curious. I'm ok leaving it out.